src/main/java/nextstep/courses/infrastructure/JdbcSessionWithNsUserRepository.java

```diff
@@ -0,0 +1,33 @@
+package nextstep.courses.infrastructure;
+
+import nextstep.courses.domain.repository.SessionWithNsUserRepository;
+import org.springframework.jdbc.core.JdbcOperations;
+
+import java.util.List;
+
+public class JdbcSessionWithNsUserRepository implements SessionWithNsUserRepository {
+
+    private JdbcOperations jdbcTemplate;
+
+    public JdbcSessionWithNsUserRepository(JdbcOperations jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    @Override
+    public boolean save(Long sessionId, Long nsUserId) {
+        String sql = "insert into session_ns_user (session_id, ns_user_id) values(? , ?)";
+        return jdbcTemplate.update(sql, sessionId, nsUserId) > 0;
+    }
```

Comment:
```suggestion
    @Override
    public boolean save(SessionWithNsUser sessionWithNsUser) {
        String sql = "insert into session_ns_user (session_id, ns_user_id) values(? , ?)";
        return jdbcTemplate.update(sql, sessionWithNsUser.getSessionId(), sessionWithNsUser.getNsUserId()) > 0;
    }
```

도메인을 받지 않고 진행하신 이유가 궁금해요~

Reply:
브릿지 테이블에서 필요한 것은 해당 도메인의 식별자만 있으면 되기 때문에 가볍게 받으려고 하였습니다.
객체를 받고 꺼내는 방법이 더 좋을까요?